src/Messages/OWSDisappearingMessagesJob.m

```diff
@@ -142,15 +154,19 @@ - (void)setExpirationForMessage:(TSMessage *)message
     [self setExpirationForMessage:message expirationStartedAt:[NSDate ows_millisecondTimeStamp]];
 }
 
-+ (void)setExpirationForMessage:(TSMessage *)message expirationStartedAt:(uint64_t)expirationStartedAt
+- (void)setExpirationForMessage:(TSMessage *)message expirationStartedAt:(uint64_t)expirationStartedAt
 {
-    dispatch_async(self.serialQueue, ^{
```

Comment:
This is a misleading diff, I think. The class method got removed by accident. I've restored it.